pywinauto/unittests/test_application.py

```diff
@@ -607,6 +607,26 @@ def test_kill_(self):
 
         self.assertRaises(AttributeError, app.UntitledNotepad.Edit)
 
+    def test_process_is_running(self):
+        """Tests process is running function"""
+        app = Application()
+        app.start(_notepad_exe())
+        self.assertTrue(app.is_process_running())
+        app.kill()
+        self.assertFalse(app.is_process_running())
+
+    def test_wait_process_exit(self):
+        """Tests wait process to exit"""
+        app = Application()
+        app.start(_notepad_exe())
+        app.UntitledNotepad.Edit.type_keys("hello")

```

Comment:
Why there is a need for a dialog, isn't it the same as waiting on the main window?Can't you just call  ```app.wait_for_process_exit```  in ```test_process_is_running``` before calling ```app.kill``` ? In this case, we can remove this test at all.